src/shared/templates/shared/partial/modal_student_details_affect.html

```diff
@@ -7,13 +7,25 @@
     <ul class="list-unstyled">
     {% for reaction in my_recent %}
         <li class="mb-0_5">
+            {% if reaction.unauthorized %}
+            <a href="#" data-cfw="tooltip" title="Not authorized to view">
+                <span class="icon-block" title="Not authorized to view"></span>
+                <strong>{{ reaction.book.title }}</strong>
+            </a>
+            {% else %}
             <a href="#" onclick="vocabCheck.start(this, '{{reaction.book.id}}'); return false;">
                 <strong>{{ reaction.book.title }}</strong>
-            </a><br>
-            <em>{{ reaction.affect_free_response }}</em>
+            </a>
+            {% endif %}
+            {% if reaction.affect_free_response and reaction.affect_free_response.strip %}
+                <br><em>{{ reaction.affect_free_response }}</em>
+            {% endif %}
         </li>
     {% endfor %}
     </ul>
+    {% if any_unauthorized_book %}
+    <p class="form-text"><em>Readings with the symbol <span class="icon-block" title="Not authorized to view"></span> were uploaded by the student.</em></p>
+    {% endif %}
```

Comment:
This looks great.  We want to also do the same thing on the main page, if this icon appears in the readings table.

Reply:
Okay.  The simplest way to proceed is to show it always, similar to how the "Assigned" legend is shown at the top.  Due to paging, some pages of results will have no assigned books listed, yet "Assigned" is always present.  The "Unauthorized" legend could behave the same way -- always shown.

Reply:
The latest push addresses your latest three comments, @bgoldowsky.

In addition, I did writing a version of the code that supports a conditional presentation of the "Unauthorized" legend at the bottom of the reading details panel.  However, I still lean towards it being displayed unconditionally, so that is the version I pushed.  I do have a copy of the conditional version and can swap it in if necessary.

Reply:
I wrote:

> In addition, I did write a version of the code that supports a conditional presentation of the "Unauthorized" legend at the bottom of the reading details table. However, I still lean towards it being displayed unconditionally, so that is the version I pushed. I do have a copy of the conditional version and can swap it in if necessary.

Rather than store that version locally on my machine, I've made another PR #369 that includes the code for showing the "unauthorized" legend only when an unauthorized book is shown in the table.  PR #369 is based on the branch for this PR.  Thus, the code is the same for the two PRs except for the conditional display of the legend.